extensions/parquet/table/plugin/channelprovider/tracked/src/main/java/io/deephaven/parquet/table/plugin/channelprovider/tracked/TrackedSeekableChannelsProvider.java

```diff
@@ -50,7 +52,9 @@ public ChannelContext makeContext() {
     public final SeekableByteChannel getReadChannel(@NotNull final ChannelContext context, @NotNull final URI uri)
             throws IOException {
         // context is unused here because it is NULL
-        return new TrackedSeekableByteChannel(fileHandleFactory.readOnlyHandleCreator, new File(uri));
+        Assert.assertion(uri.getScheme() == null || uri.getScheme().equals(ParquetFileReader.FILE_URI_SCHEME),
```

Comment:
FILE_URI_SCHEME should move to this package.

Reply:
I have added FILE_URI_SCHEME to this class, but I cannot remove it from ParquetFileReader because there are some checks and assertions in the code around file schemes.